slicer/SemiologyVisualization.py

```diff
@@ -37,10 +37,10 @@ def __init__(self, parent):
     self.parent.categories = ["Epilepsy Semiology"]
     self.parent.dependencies = []
     self.parent.contributors = [
-      "Fernando Perez-Garcia (3D Slicer Module and data processing)",
-      "Ali Alim-Marvasti (data collection and processing modules)",
-      "Gloria Romagnoli (data collection)",
-      "John Duncan (supervisor)",
+      "Fernando Perez-Garcia (SVT software: Design, 3D Slicer Module, Tests, and GUI API)",
```

Comment:
I don't think it's fair to say that all I've done related to `mega_analysis`/"MegaAnalysis Preprocessing Modules" is "Tests"...

Reply:
Another option, maybe easier, is just to not specify what each of us has done.